### DISCUSSION TITLE
Add Workflow to custom package

### DISCUSSION THREAD

**Discussion by ablot** [bonsai-rx/bonsai#1533] [Q&A] [answered]
Hi all,

We are using a simple custom package created following the [doc here](https://bonsai-rx.org/docs/articles/create-package.html) and it does work for source/transform/sink. However the workflow that I add don't appear in the Toolbox.

I use Add/New Item... and select Workflow in visual studio, rebuild the solution, re-install it in a local bonsai install and nothing appears. Is there another configuration step to add workflows?

Thanks!

**Comment by bruno-f-cruz** [comment, marked as answer]
You might need to tag it as a 'Resource'. Can you try that?

**Comment by ablot** [comment]
Indeed that worked! 

(added 
```
  <ItemGroup>
    <EmbeddedResource Include="**\*.bonsai" />
  </ItemGroup>
 ```
in the .csproj file of my package)
